From 25fb14174a44600dd2ba53b5099d64580886aca1 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Mon, 5 Feb 2024 22:04:23 -0600
Subject: [PATCH] ftbfs glibc 2.39

The conftests for with-unix-stdio-64 broke with glibc v2.39, and using --with-unix-stdio-64=yes sets these to the wrong function. 

Force override them to the ones that would have been used had the conftests worked.

---
 gdal/m4/acinclude.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gdal/m4/acinclude.m4 b/gdal/m4/acinclude.m4
index 139e600951..fa2dbfcd44 100644
--- a/gdal/m4/acinclude.m4
+++ b/gdal/m4/acinclude.m4
@@ -122,8 +122,8 @@ AC_DEFUN([AC_UNIX_STDIO_64],
 
   if test x"$HAVE_UNIX_STDIO_64" = x"yes" ; then
     HAVE_UNIX_STDIO_64=yes
-    VSI_FTELL64=ftell64
-    VSI_FSEEK64=fseek64
+    VSI_FTELL64=ftello64
+    VSI_FSEEK64=fseeko64
   fi
 
   if test x"$HAVE_UNIX_STDIO_64" = x"" ; then
-- 
2.43.0

